.github/workflows/commit.yml

```diff
@@ -72,26 +89,39 @@ jobs:
         run: task init
       - name: Build the goat
         run: task build
-      - name: Bump the pre-commit revision
+      - name: Bump the version
+        id: bump-version
         run: |
           git config --global user.name 'Seiso Automation'
           git config --global user.email 'automation@seisollc.com'
           # Non-string variables are pending https://github.com/go-task/task/issues/140
           task pre-commit-update -- commit
+          task release
```

Comment:
We should add to this step to point the submodule back to main and pull the latest

Reply:
Did this